packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -24,8 +130,112 @@ class DeepLinksScreen extends Screen {
 
   @override
   Widget build(BuildContext context) {
-    return const Center(
-      child: Text('TODO: build deep link validation tool'),
+    return const DeepLinkPage();
+  }
+}
+
+class DeepLinkPage extends StatefulWidget {
+  const DeepLinkPage({super.key});
+
+  @override
+  State<DeepLinkPage> createState() => _DeepLinkPageState();
+}
+
+class _DeepLinkPageState extends State<DeepLinkPage> {
+  String responseBody = 'empty response';
+  int? selectedRowIndex;
+  String searchContent = '';
+  bool bundleByDomain = false;
+
+  void setBundleByDomain(bool shouldBundleByDomain) {
+    setState(() {
+      bundleByDomain = shouldBundleByDomain;
+    });
+  }
+
+  void setResponse(String response) {
+    setState(() {
+      responseBody = response;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
```

Comment:
use a column instead and wrap the data table in an Expanded widget

Reply:
The data table can be longer than the screen height.

Reply:
Do we really want the controls to scroll out of view if the data table is large though? I suggest using a column and then making the data table itself scrollable so that we do not lose the controls to scroll through the table. You can see an example of this on other DevTools pages like the logging page, for example, where the controls at the top of the screen are fixed and only the table scrolls.